### PR TITLE
Update FAQ and Config with announcements info and other fixes

### PR DESCRIPTION
**Pull request by keybits**
- add info about subscribing to: https://discuss.circleci.com/c/announcements
- fix formatting 
- update info on chat application notifications

### CODE REVIEW

File: jekyll/_docs/faq.md [hunk 1]
```diff
@@ -6,40 +6,61 @@ categories: [getting-started]
...
+You may also like to follow our **[changelog](https://circleci.com/changelog/)** to see what has been released.
+
+Our **[status page](http://status.circleci.com/)** gives information on current system status, outages and past incidents.
+
 ## Authentication
 
 ### Do you support BitBucket or GitLab?
-Currently we support authentication with GitHub and Bitbucket. We do look at feature requests on [Discuss](https://discuss.circleci.com/) if you want to show support for other version control systems.
+
+Currently we support authentication with GitHub and Bitbucket. Please vote on relevant [Feature REquest](https://discuss.circleci.com/c/feature-request) if you want to show support for other version control systems.
```
Comment (felicianotech) [inline jekyll/_docs/faq.md]: Feature REquest -> Feature Request

File: jekyll/_docs/faq.md [hunk 2]
```diff
@@ -6,40 +6,61 @@ categories: [getting-started]
...
 
 ### I can’t give CircleCI the access to all my private repositories. What do I do?
+
+Please see our guides and explanations about [GitHub and Privacy](https://circleci.com/docs/github-privacy/)
```
Comment (felicianotech) [inline jekyll/_docs/faq.md]: We shouldn't link https://circleci.com/docs/github-privacy/ here as this guide/page is going away shortly (the whole category it's in actually is going away).

File: jekyll/_docs/faq.md [hunk 3]
```diff
@@ -6,40 +6,61 @@ categories: [getting-started]
...
-[changelog](https://circleci.com/changelog/) to be notified as soon as
-this feature is available.
+
+### Can I get notifications to team chat applications?
+
+CircleCI supports Slack, HipChat, Campfire, Flowdock and IRC notifications; you configure these notifications from your project’s **Project Settings > Notifications** page.
+
+### Can I send Slack / HipChat / IRC notifications for specific branches only?
+
+There is [experimental support for per branch chat notifications](https://circleci.com/docs/configuration/#per-branch-notifications).
```
Comment (felicianotech) [inline jekyll/_docs/faq.md]: [experimental support for per branch chat notifications](https://circleci.com/docs/configuration/#per-branch-notifications) => [experimental support for per branch chat notifications]({{site.baseurl}}/configuration/#per-branch-notifications)

**Comment by keybits** [comment]
Thanks for the suggestions @felicianotech 

I've updated the PR accordingly.

File: jekyll/_docs/faq.md [hunk 4]
```diff
@@ -6,40 +6,59 @@ categories: [getting-started]
...
+Currently we support authentication with GitHub and Bitbucket. Please vote on relevant [Feature Request](https://discuss.circleci.com/c/feature-request) if you want to show support for other version control systems.
 
 ### I can’t give CircleCI the access to all my private repositories. What do I do?
+
 GitHub has only recently added the fine-grained permissions options, and
 we are still working on supporting them.
 
-In the meantime, the suggested workaround is to create an additional
-user on GitHub with a limited set of permissions and use that account to
+In the meantime, the [suggested workaround](https://circleci.com/docs/github-security-ssh-keys/) is to create an additional user on GitHub with a limited set of permissions and use that account to
```
Comment (felicianotech) [inline jekyll/_docs/faq.md]: Not a relative link.

[suggested workaround]({{site.baseurl}}/github-security-ssh-keys/)

**Comment by felicianotech** [comment]
@keybits Made one more comment. Otherwise LGTM.

**Comment by keybits** [comment]
@felicianotech Glad you've got eagle eyes spotting those errors! Thank you :-)

**Comment by felicianotech** [comment]
@keybits I'm glad you're so willing to improve docs. Thanks so much.

**Pull request merged**